rfc/guides-comments.md

```diff
@@ -0,0 +1,75 @@
+Options available for adding comments to the deployment guides are as follows:
+
+1.  Disqus: <https://disqus.com/>
+
+Really good third party service and the most popular. It has 3 options:
+
+-   Basic: This has ad's (not recommended)
+
+-   Plus $9 per month (Suitable for under 50k visitors daily)
+
+-   Pro $89 per month (Suitable for 150k visitors daily)
+
+Pros:
+
+-   It provides a dashboard to manage for all the comments posted.
+
+-   Analytics available for comments on the dashboard.
+
+-   Very easy to install
+
+Cons
+
+-   Being a third party service, we don't know what our comments become and how disqus uses the data retrieved from the comments 
+
+-   Disqus doesn't have a great track record on privacy
+
+-   We cannot control the look and feel of the design
+
+-   Does not allow anonymous content
```

Comment:
That's probably OK, as it likely reduces spam. That said, I still get a few spam posts per week on my blog, so it's not perfect.